### ISSUE TITLE
.text() should return the text when an element exists

### ISSUE THREAD

**Issue by sextondb-gus** [getgauge/taiko#1066]
### Expected behavior
.text() should return the text when an element exists

### Actual behavior
the .text() promise resolves with an undefined value

### Steps to reproduce
```
const { openBrowser, goto, text, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
	await goto("https://taiko.gauge.org/");
        const textExists = await text("Reliable browser automation").exists();
	const textValue = await text("Reliable browser automation").text();
	console.log(`exists: ${textExists}`);
	console.log(`value: ${textValue}`);
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();

//exists: true
//value: undefined

```

### Versions
```
node -v: v13.1.0
taiko: 1.0.3 (via npm), and master (17cd78cec3345ba870e86766cd3307fd71b5f5ab via github)
```

**Comment by BugDiver** [comment]
@sextondb-gus I can replicate it. We will try to fix it asap.

**Comment by sextondb-gus** [comment]
Is there any additional information I can provide to help this issue?
Is there a work around I can use to get the text of an ElementWrapper or ElementWrapperList until the issue is resolved?

**Comment by negiDharmendra** [comment]
@sextondb-gus You can use `evaluate` API to get the text untill we fix this issue. 
**Example:**
`await evaluate(text("Reliable browser automation"), e => e.wholeText)`

**Comment by sextondb-gus** [comment]
@negiDharmendra  Thanks!

**Comment by saikrishna321** [comment]
Fixed version `taiko 1.0.5`
